vendor/google.golang.org/protobuf/encoding/protowire/wire.go

```diff
@@ -29,12 +29,8 @@ const (
 )
 
 // IsValid reports whether the field number is semantically valid.
-//
-// Note that while numbers within the reserved range are semantically invalid,
-// they are syntactically valid in the wire format.
-// Implementations may treat records with reserved field numbers as unknown.
 func (n Number) IsValid() bool {
-	return MinValidNumber <= n && n < FirstReservedNumber || LastReservedNumber < n && n <= MaxValidNumber
```

Comment:
it looks like this was the only documented breaking change in the 1.28 → 1.30 range ... I swept and didn't find any use of this that would be impacted by this change